### ISSUE TITLE
Testing repository

### ISSUE THREAD

**Issue by crackerizer** [cubieplayer/Cubian#220]
Hello,

I'd like to help creating and maintaining the testing repository for cubian.
Could it be possible?

**Comment by cubieplayer** [comment]
Yes, [cubian's repository](https://github.com/cubieplayer/packages.cubian.org.) is maintained using reprepro and hosted by github.
If you are a serious developer, contributor, I'd like to give you the access to the repository.
Please read https://github.com/cubieplayer/Cubian/wiki/Contribute-deb-package first.
You can contact me directly in this thread.

**Comment by crackerizer** [comment]
I'm new to repository maintenance, correct me if i'm wrong.
do i have to create the entire repository myself or just create the testing branch from master and do a pull request?
